http/server/api.go

```diff
@@ -48,7 +48,11 @@ func (s *rulesetAPI) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 			return
 		}
 
-		s.create(w, r, path)
+		if len(path) != 0 {
+			w.WriteHeader(http.StatusNotFound)
```

Comment:
Isn't returning http.StatusBadRequest more relevant here?

Reply:
No because that's the wrong path: `/rulesets/p/a/t/h` and `/rulesets/` are different paths